install.sh

```diff
@@ -29,3 +29,16 @@ apt-get install -y build-essential curl git imagemagick libgdal-dev libgeos-dev\
   vim zlib1g-dev
 
 gem install bundler
+
+curl -sL https://deb.nodesource.com/setup_4.x | bash
+apt-get install -y nodejs
+add-apt-repository ppa:fcwu-tw/ppa
+apt-get update
+apt-get install -y vim
+add-apt-repository ppa:git-core/ppa
+apt-get update
+apt-get install -y git
+
+# comment out to mount via NFS outside VM:
+apt-get install -y nfs-kernel-server
```

Comment:
you can include this dependency install with the other dependencies (line 22). 
if anyone doesn't want to mount via this feature, it can still live with the installed package.
you can add info comment before the line 22 like
\# packages tip: 
\# nfs-kernel-server:  used for NFS mounting ...etc